jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>might this be related to transforming the data previously to a dense representation? just wondering? will subsampling lead to comparible results or would you recommend running this on a cluster then?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>It's comparable and it will take ages if ran on a big dataset without subsampling.</p>

Reply:
<p>Okay?</p>

Reply:
<p>" In some cases, it is requires to do sub-sampling of the data in order to avoid memory limitations"</p><p> </p><p>-&gt;</p><p> </p><p> In some cases, it is required tosub-sampling the data in order to avoid memory limitations</p>